File: Modul 2/Python/PRAK205-2310817220025-FirdaKhoirunisa.py
import math

print("Input")
a,b = map(int, input().split())

alas = math.sqrt(pow(b, 2)-pow(a, 2))
tinggi = a
keliling = a+b+alas
luas = 1/2*alas*tinggi

print("\nOutput")
print("Alas =", int(alas), "cm")
print("Tinggi =", int(tinggi), "cm")
print("Keliling =", int(keliling), "cm")
print("Luas =", int(luas), "cm^2")

File: Modul 2/C/PRAK205-2310817220025-FirdaKhoirunisa.c
#include<stdio.h>
#include<math.h>

void main(){
    int a,b;

    printf("Input\n");
    printf("");
    scanf("%d %d", &a, &b);

    int alas = sqrt(pow(b, 2)-pow(a, 2));
    int tinggi = a;
    int keliling = a+b+alas;
    int luas = alas*tinggi/2;

    printf("\nOutput\n");
    printf("Alas = %d cm\n", alas);
    printf("Tinggi = %d cm\n", tinggi);
    printf("Keliling = %d cm\n", keliling);
    printf("Luas = %d cm^2", luas);
}